scripts/release_candidate.py

```
@@ -3589,12 +3588,10 @@ def _validate_windows_setup_pe(path: Path) -> tuple[str, bool]:
         raise CandidateError("Windows Setup lacks a PE security-directory entry")
     certificate_offset, certificate_size = struct.unpack_from("<II", payload, optional_offset + 112 + 4 * 8)
     if certificate_offset == 0 and certificate_size == 0:
-        embedded_signature_present = False
+        raise CandidateError("Windows Setup lacks an embedded Authenticode signature")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)